unearthed-arcana/2022/20220308.xml

```diff
@@ -0,0 +1,1130 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<name>Unearthed Arcana: Heroes of Krynn</name>
+		<update version="0.0.1">
+			<file name="20220308.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/unearthed-arcana/2022/20220308.xml" />
+		</update>
+	</info>
+
+	<element name="Unearthed Arcana: Heroes of Krynn" type="Source" source="Core" id="ID_WOTC_SOURCE_UNEARTHED_ARCANA_20220308">
+		<description>
+			<p>In today’s Unearthed Arcana, we explore character options from the Dragonlance setting. This playtest document presents the kender race, the Lunar Magic sorcerer subclass, the Knight of Solamnia and Mage of High Sorcery backgrounds, and a collection of new feats, all for use in Dungeons &amp; Dragons.</p>
+		</description>
+		<setters>
+			<set name="abbreviation">UA20220308</set>
+			<set name="url">https://media.wizards.com/2022/dnd/downloads/UA2022HeroesofKrynn.pdf</set>
```

Comment:
should be link to the article, rather than pdf
```suggestion
			<set name="url">https://dnd.wizards.com/articles/unearthed-arcana/heroes-krynn</set>
```